client/landing/stepper/declarative-flow/trial-wooexpress-flow.ts

```diff
@@ -76,40 +73,18 @@ const wooexpress: Flow = {
 			} catch {}
 		}
 
-		const getStartUrl = () => {
-			let hasFlowParams = false;
-			const flowParams = new URLSearchParams();
-			const queryParams = new URLSearchParams();
-
-			if ( vendorId ) {
-				queryParams.set( 'vid', vendorId );
-			}
-
-			if ( aff ) {
```

Comment:
the `aff` and `vid` params aren't handled in `login()` so I'm guessing they're deprecated?

Reply:
Nope 😅  I thought they were appended to redirect_to. Just fixed it in b8395594cd8e24c43ea38cb62e00b2601f77d25d